ibis/backends/pandas/execution/strings.py

```diff
@@ -236,15 +236,16 @@ def execute_group_concat_series_gb(
 def execute_group_concat_series_gb_mask(
     op, data, sep, mask, aggcontext=None, **kwargs
 ):
-    def method(series, sep=sep):
-        return sep.join(series.values.astype(str))
+    def method(series, mask, sep=sep):
+        if mask.any():
```

Comment:
I think we can simply check if the series is empty here instead of incurring the overhead of looking at every `mask` element.

Reply:
Done